src/accessibility.md.njk

```diff
@@ -8,7 +8,11 @@ layout: layout-single-page-prose.njk
 
 The GOV.UK Design System team works hard to ensure that this Design System and GOV.UK Frontend, the codebase it uses, are accessible.
 
-When the Design System went into public beta in June 2018, it had been audited by accessibility specialists [DAC (Digital Accessibility Centre)](https://digitalaccessibilitycentre.org/) and met level AA of the Web Content Accessibility Guidelines (WCAG 2.0).
+This Design System and GOV.UK Frontend were last tested for accessibility in May 2019. The test was carried out by accessibility specialists [DAC (Digital Accessibility Centre)](https://digitalaccessibilitycentre.org/) against level AA of the Web Content Accessibility Guidelines (WCAG) 2.1.
+
+You can see the [issues DAC raised and what we fixed](https://github.com/issues?q=is%3Aissue+archived%3Afalse+org%3Aalphagov+label%3A%22audit%3A+may-2019%22+).
+
+We know there are a small number of accessibility issues that need to be resolved. You can see [a list of outstanding issues](https://github.com/issues?q=is%3Aissue+archived%3Afalse+org%3Aalphagov+label%3A%22audit%3A+may-2019%22+is%3Aopen).
```

Comment:
2i noted that this used passive tense, and the link could flow into the sentence a bit more cleanly. Suggestion: 

```suggestion
There are a small number of [accessibility issues that we still need to fix](https://github.com/issues?q=is%3Aissue+archived%3Afalse+org%3Aalphagov+label%3A%22audit%3A+may-2019%22+is%3Aopen).
```